doc/install/install.rst

```diff
@@ -1,224 +1,37 @@
-Installation guide
-==================
-
-These instructions provide setup guidance for both Linux (tested on Ubuntu
-22.04.4 LTS) and macOS. While most steps are similar across platforms, some
-adjustments may be needed depending on your specific OS version or
-configuration.
-
-.. note::
-
-    - These instructions assume you are using the bash shell. If you use a
-      different shell (e.g., zsh, fish), commands may require slight
-      modifications.
-    - Some steps require administrative privileges and the use of sudo. If you
-      do not have such privileges, please consult your system administrator.
+OpenSn Installation
+===================
 
 Install Development Tools
 -------------------------
 
 The following packages are required for **OpenSn** installation and development:
 
-1. A recent version of ``clang++``/``g++`` that supports C++17
-2. ``gfortran`` (required by the BLAS component of PETSc)
-3. ``flex`` and ``bison`` (required by the PTSCOTCH component of PETSc)
-4. Python3 v3.9+ and ``pip`` (required by PETSc and OpenSn)
-5. Git version control system
-6. CMake v3.12+
-7. MPI (OpenMPI, MPICH, and MVAPICH have been tested)
-8. HDF5
-9. Doxygen (required for generating the OpenSn documentation)
+1. A recent version of ``clang++``/``g++`` that supports C++20
+2. ``flex`` (required by the PTSCOTCH component of PETSc)
```

Comment:
`bison` is still needed, see [this](https://gitlab.com/petsc/petsc/-/blob/main/config/BuildSystem/config/packages/PTScotch.py#L40-41)

Reply:
Instead of adding `bison` as a system dependency, it's installed with `PETSc`.

Reply:
Yeah. I missed that. 👍